tests/foreman/ui/test_discoveredhost.py

```diff
@@ -475,7 +485,7 @@ def test_positive_provision_from_facts(self):
 
     @run_only_on('sat')
     @tier3
-    def test_positive_delete(self):
+    def test_positive_delete_1(self):
```

Comment:
Is `test_positive_delete_1` intended change? Imo `test_positive_delete` was good enough and basically we agreed test names with numbers in the end are bad practice and introduced test naming conventions long time ago.